lib/client.js

```diff
@@ -12,6 +12,21 @@ var Client = function (config) {
 		config.baseUrl = "https://api.catapult.inetwork.com";
 	}
 
+	function handleResponse(response) {
```

Comment:
The linter was complaining about using the function before it was defined, and it was breaking build. Moving the declarations of the helper functions about `handleResponse` fixed build.